tools/embark-oss/README.md

```diff
@@ -0,0 +1,21 @@
+# Embark OSS tool
+
+This tool is used to validate that our open source projects adhere to our open source guidelines.
```

Comment:
```suggestion
This tool is used to validate that our open source projects adhere to open source guidelines.
```

Reply:
They are our guidelines rather than some general open source guidelines, so it wouldn't be correct to remove this I think.

Reply:
+1 to what Louis said